src/bindings/python/src/pyopenvino/graph/node_output.hpp

```diff
@@ -78,6 +78,7 @@ void regclass_graph_Output(py::module m, std::string typestring)
                )");
     output.def("get_shape",
                &ov::Output<VT>::get_shape,
+               py::return_value_policy::copy,
                R"(
                 The shape of the output referred to by this output handle.
```

Comment:
In my opinion, there should be a note about creating a copy of the Shape object. As for example `get_tensor` or `get_rt_info` are returning references -- so retrun-by-copy without the explicit note could create a misunderstanding in general.

CC @akuporos what do you think?

Reply:
I added a note about copy below. But anyway, c++ api also doesn't allow to change port's shape directly and i believe copying in this case is not heavy and for python user it shouldn't have a big difference here.

Reply:
Sure, copy is fine here. I was only questioning if the note is necessary here, as for the user a possibility to change something could mean that the object reflects an actual state of internal values.